packages/web/src/pages/profile-page/components/desktop/ProfileMutuals.tsx

```diff
@@ -64,7 +50,7 @@ export const ProfileMutuals = () => {
     dispatch(setVisibility(true))
   }, [dispatch, userId])
 
-  if (!profile || userId === accountId || mutuals.length === 0) {
+  if (!profile || userId === accountId || !mutuals) {
```

Comment:
do we want to render an empty array now?

Reply:
nice call, changing